requirements.txt

```diff
@@ -1,4 +1,12 @@
 # To install these Python dependencies, please type:
 # pip install -r requirements.txt
 
-streamlit
+python>=3.6.6
+streamlit>=0.45
+numpy>=1.17
+pandas>=0.25
+requests>=2.22
+bokeh>=1.3.4
+# plotly==3.10
+# plotly_express==0.3.1
+statsmodels>=0.10
```

Comment:
is requirements up to date? e.g., I don't see pmdarima. I haven't tried running this myself (though that's not a requirement)

Reply:
Updated `requirements.txt`.